src/trove_classifiers/__init__.py

```diff
@@ -315,6 +315,7 @@
     "License :: OSI Approved :: Jabber Open Source License",
     "License :: OSI Approved :: MIT License",
     "License :: OSI Approved :: MIT No Attribution License (MIT-0)",
+    "License :: OSI Approved :: MIT-CMU License",
```

Comment:
This probably should have included the SPDX identifier in parentheses like the rest of our license classifiers. OSI also calls this the "CMU License" instead: https://opensource.org/license/cmu-license

```suggestion
    "License :: OSI Approved :: CMU License (MIT-CMU)",
```

Reply:
Thanks, I've opened https://github.com/pypa/trove-classifiers/pull/190 as a follow-up.